extensions/sql-migration/src/models/loginMigrationModel.ts

```diff
@@ -179,7 +180,25 @@ export class LoginMigrationModel {
 			}
 		}
 
-		this.errorCountMap.set(LoginMigrationStep[step], errorBuckets);
+		// Making a string of the map elements of errorBuckets
+		var errorBucketsString = JSON.stringify(
+			Array.from(errorBuckets.entries()).reduce((o: any, [key, value]) => {
+				o[key] = value;
+				return o;
+			}, {})
+		);
+
+		// Retaining this step in case a revert is needed, but we will be using errorCountMapString for telemetry
+		this.errorCountMap.set(LoginMigrationStep[step], errorBucketsString);
+
+		// Creating an array of the error codes and its counts for each step
+		this.errorCountMapString.push(
+			"[" +
+			LoginMigrationStep[step].toString() +
+			":" +
+			errorBucketsString.toString() +
```

Comment:
aren't both LoginMigrationSetep[step] and errorBucketString string?  .toString() should not be required.

Reply:
removed toString()